### PR TITLE
SpatialEditorGizmo add_mesh bindings fix

### PR DESCRIPTION
**Pull request by SaracenOne**
Fixes an error when trying to use generated api.json to generate GDNative bindings.

### CODE REVIEW

**Comment by akien-mga** [comment]
You should not commit `api.json`.

**Comment by SaracenOne** [comment]
My bad, that shouldn't have been in there 👀
I'll fix it now

File: editor/spatial_editor_gizmos.cpp [hunk 1]
```diff
@@ -729,7 +729,7 @@ void EditorSpatialGizmo::set_plugin(EditorSpatialGizmoPlugin *p_plugin) {
 void EditorSpatialGizmo::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("add_lines", "lines", "material", "billboard"), &EditorSpatialGizmo::add_lines, DEFVAL(false));
-	ClassDB::bind_method(D_METHOD("add_mesh", "mesh", "billboard", "skeleton", "material"), &EditorSpatialGizmo::add_mesh, DEFVAL(false), DEFVAL(Variant()), DEFVAL(Variant()));
+	ClassDB::bind_method(D_METHOD("add_mesh", "mesh", "billboard", "skeleton", "material"), &EditorSpatialGizmo::add_mesh, DEFVAL(false), DEFVAL(Ref<SkinReference>()), DEFVAL(Variant()));
```
Comment (akien-mga) [inline editor/spatial_editor_gizmos.cpp]: Shouldn't the other default argument be `Ref<Material>()` in a similar way?

**Comment by akien-mga** [comment]
Thanks!

**Comment by neikeq** [comment]
Does GDNative's api.json generator not understand `Variant()` (NIL) when it comes to Reference parameters?

**Pull request merged**